modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase

```diff
@@ -17,7 +17,7 @@ definition {
 	}
 
 	@ignore = "Test Stub"
-	@priority = "5"
+	@priority = 5
```

Comment:
Minor nitpick but prob better to include fixing typos like this in the original commit to keep the commit history clean.